felix/nftables/actions.go

```diff
@@ -263,7 +263,7 @@ type LogAction struct {
 }
 
 func (g LogAction) ToFragment(features *environment.Features) string {
-	return fmt.Sprintf(`log prefix "%s " level info`, g.Prefix)
+	return fmt.Sprintf(`log prefix "%s: " level info`, g.Prefix)
```

Comment:
This change is basically to align nftables with iptables: https://github.com/projectcalico/calico/blob/9070db0fed5adbdc8bfdc9e391cae38ca4f6bc99/felix/iptables/actions.go#L224